### ISSUE TITLE
Equipset system is out of era

### ISSUE THREAD

**Issue by TracentEden** [EdenServer/community#4721]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details
Equipset system is out of era and thus could be disabled or removed. I think the system was not introduced until 2014-2015 (see https://forum.square-enix.com/ffxi/threads/43454-dev1223-Macro-Editing-Menu-Addition-Equipment-Sets).

**Comment by Spintimalli** [comment]
Merry Christmas, 
Love Satan

**Comment by Steam-arch** [comment]
You forgot mog wardrobe  1/2/3/4

**Comment by KnowOne134** [comment]
Except that's a listed change that on purpose

**Comment by TracentEden** [comment]
Yeah, I mean if intention is to keep for QoL then I guess add to that list of changes at https://edenxi.com/about , but I didn't find it there.

**Comment by KnowOne134** [comment]
I was talking of the wardrobes. They are listed

**Comment by Steam-arch** [comment]
Yep. Literally no point in forcing people to become autobots and use ashita cast. Unless.. we disallow both, would be a refreshing change 👁️

**Comment by TracentEden** [comment]
Ah yeah, was just replying to steam
